places/ChrisH75-hometown.md

```diff
@@ -0,0 +1,7 @@
+#Columbus, GA
+
+## Great Places to Eat
+
+## Fun things to do
```

Comment:
@ChrisH75 Do residents have to drive to Atlanta or other nearby larger cities in order to see big-name concerts? Or is Columbus large enough to draw a lot of music shows on its own?